app/models/column_data_type.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+
+class ColumnDataType(models.Model):
+    display_name = models.TextField()
+    description = models.TextField()
+
+
+"""
+COLUMN_DATA_TYPES contains all the "sprout" types.
+
+Changing the properties of the content can be risky. See the list below
+
+- NO RISK: The `description` is safe to change
+- LITTLE RISK: The `display_name` should rarely change. Changes will confuse the user and
+  should never change to a different type (Text --> Decimal)
+- FORBIDDEN: id. You should never change the id
+"""
+COLUMN_DATA_TYPES = [
+    ColumnDataType(
+        id=0,
+        display_name="Decimal",
+        description="Also known as a Float or Double Precision. This field stores decimal numbers. Use this for items like height, blood glucose, or other measurements with high degrees of precision",
+    ),
+    ColumnDataType(
+        id=1,
+        display_name="Whole Number",
+        description="Also known as an `integer`. This field only allows whole numbers to be stored. Use this when you want to store data that doesn't need a decimal (e.g. number of people in a household)",
+    ),
+    ColumnDataType(
+        id=2,
+        display_name="Text",
+        description="A text field without a limit to how many characters can be stored",
+    ),
+    ColumnDataType(
+        id=3,
+        display_name="Yes/No",
+        description="Also known as Yes/No, Boolean, or Logical. This stores data as either 0/1 or True/False. If you need/want to store actual words, use the Text field",
```

Comment:
```suggestion
        description="Also known as Boolean, or Logical. This stores data as either 0/1 or True/False. If you need/want to store actual words, use the Text field",
```
"Yes/No also known as "Yes/No"